sample_app/sampleapp_common.c

```diff
@@ -922,23 +1177,26 @@ void app_plug_dap (pnet_t * net, void * arg)
       PNET_SLOT_DAP_IDENT,
       PNET_SUBMOD_DAP_IDENT,
       PNET_MOD_DAP_IDENT,
-      PNET_SUBMOD_DAP_IDENT);
+      PNET_SUBMOD_DAP_IDENT,
+      &cfg_dap_data);
```

Comment:
What is cfg_dap_data? Is it a global variable?

Reply:
Done, moved global constant into this function and added some comments on why it is needed